File: docs/legacy(old PACE)/C#/4.0/PaceAPISample/sample/ReadGLAccount.cs
using System;

using efipaceservices;

namespace Pace_Web_Service_SDK
{
	/// <summary>
	/// Summary description for ReadGLAccount.
	/// </summary>
	public class ReadGLAccount
	{
        public static void Run()
        {
            int accountID = 1;

            ReadObjectHttpBinding readObjectHttpBinding = PaceClient.getReadObjectHttpBinding();

            GLAccount acct = new GLAccount();
            acct.id = accountID;
            try
            {
                acct = readObjectHttpBinding.readGLAccount(acct);
                Console.WriteLine("The account balance is " + acct.currentPeriodBalance);
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
            }
        }
	}
}

File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/sample/ReadGLAccount.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class ReadGLAccount
    Public Shared Sub Run()

        Dim accountID As Integer = 1

        Dim readObjectHttpBinding As ReadObjectHttpBinding = PaceClient.getReadObjectHttpBinding()

        Dim acct As GLAccount = New GLAccount()
        acct.id = accountID
        Try
            acct = readObjectHttpBinding.readGLAccount(acct)
            Console.WriteLine("The account balance is " + acct.currentPeriodBalance.ToString)
        Catch ex As Exception
            Console.WriteLine(ex.Message)
        End Try
    End Sub
End Class
